.github/workflows/update_credential_provider_script.yml

```diff
@@ -0,0 +1,37 @@
+name: Update credential provider script
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.7]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r ./test-fixtures/requirements.txt
+    - name: Modify bitrise.yml 
+      run: |
+        python ./Assets/update.py
```

Comment:
I think the name does not match with the python script... should it be CC_Python_Update.py?

Reply:
Ah true